hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/diskUsage/diskUsage.tsx

```diff
@@ -409,30 +422,44 @@ export class DiskUsage extends React.Component<Record<string, object>, IDUState>
                 {(duResponse.size > 0) ?
                   ((duResponse.size > 0 && duResponse.subPathCount === 0) ?
                     <div style={{height: 800}}>
+                    <div  className='metadatainformation'>
                       <br/> {' '}
-                      <h3>This object is a key with a file size of {this.byteToSize(duResponse.size, 1)}.<br/> {' '}
                         You can also view its metadata details by clicking the top right button.
-                      </h3>
-                    </div> :
+                    </div>
```

Comment:
I think we could remove this as well, as the plan with this patch was to default fix the metadata details on the right side and with the button you could close it.

Reply:
Could not Understand which Line  text need to remove siyao suggested to keep text.

Reply:
I can't suggest changes here, I thought to remove the whole div that starts on line 425.

Reply:
Ditto Zita. We can safely remove the second line as well.

Reply:
Done in ae1ff1d42a30e5e15a52946f99eed804f9615f5e